ui/src/containers/Owner/CreatedAt.js

```diff
@@ -2,5 +2,5 @@ import moment from 'moment';
 
 export default (createdAt) => {
   if (!createdAt) return '';
-  return `Created ${moment(createdAt).fromNow()} - ${moment(createdAt).format('YYYY-MM-DD HH:mm:ss')}`;
+  return `Created ${moment(createdAt).fromNow()}`;
```

Comment:
Has the full date and time been moved to a tooltip? I think that's actually really useful sometimes.

Reply:
Agreed - we shouldn't be redacting any information the client/user already has access to. Add as a "tooltip" hover element.

Reply:
@happy-machine this PR removes the full date from the text from the model list items that tells the user when models (like visualisations) were created. It still has the relative date using `moment(createdAt).fromNow()`, but sometimes the full date is also useful especially for us debugging issues. It's unclear in this pull request if the full date has been completely removed from the UI or moved into a tooltip on the model list item. I don't currently have time to look into this so if you do get some time @happy-machine please do take a look, I'd suggest that if it hasn't been added to the tooltip we just revert this change for now instead of trying to add the tooltip to save some time and risk.